### PR TITLE
Added developer documentation and improved inline API documentation

### PR DESCRIPTION
**Pull request by vdboor**

### CODE REVIEW

File: src/rest_framework_dso/crs.py [hunk 1]
```diff
@@ -16,10 +19,21 @@
...
+#: European Terrestrial Reference System 1989
+ETRS89 = CRS.from_string("EPSG:4258")
+
+#: The default suggested CRS (e.g for use in WFS)
 DEFAULT_CRS = RD_NEW
+
+#: Other suggested CRS's (e.g for use in WFS)
 OTHER_CRS = [WGS84, WEB_MERCATOR, ETRS89]
+
+#: All coordinate reference systems exposes by this file.
```
Comment (lbam) [inline src/rest_framework_dso/crs.py:38]: Exposes => exposed.

**Review by lbam — APPROVED** [review]

File: src/rest_framework_dso/crs.py (hunk 1 above)
Comment (vdboor) [reply src/rest_framework_dso/crs.py]: tx!

**Review by jjmurre — APPROVED** [review]
Nice!

**Review by datapunt — APPROVED** [review]
:+1:

**Pull request merged**